vendors/espressif/boards/esp32/aws_demos/application_code/main.c

```diff
@@ -185,11 +185,6 @@ static void prvMiscInitialization( void )
 
             xRet = esp_bt_controller_mem_release( ESP_BT_MODE_BLE );
 
-            if( xRet == ESP_OK )
-            {
-                xRet = esp_bt_controller_mem_release( ESP_BT_MODE_BTDM );
-            }
```

Comment:
This is removed because, controller memory for classic BT is already freed in case of NimBLE. So, we just need to free controller memory for BLE in case of cleanup.